start.php

```diff
@@ -31,11 +31,36 @@
 
 	global $fs_active_plugins;
 
-	$this_sdk_relative_path = plugin_basename( dirname( __FILE__ ) );
+    $paths = array(

```

Comment:
I don't see the value of putting the paths in a `$paths` variable, instead of having separated variables.